fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -4996,10 +5001,9 @@ class FMCMainDisplay extends BaseAirliners {
     //TODO: Can this be util?
     getGW() {
         let fmGW = 0;
-        if (this.isAnEngineOn() && isFinite(this.zeroFuelWeight)) {
+        //Simplified to just checking fuelWeight as GetFOB handles what fuel level to use (block vs tank reading)
+        if (isFinite(this.zeroFuelWeight)) {
```

Comment:
Careful, what happens if you've not entered a block fuel yet?

Reply:
Hmm Yes Thanks for catching 

 Can the way the current aircraft release handles this scenario be used as a "control" or do the IRL pilots need to be consulted for what the behavior is in this situation?

Reply:
> Hmm Yes Thanks for catching 
> 
>  Can the way the current aircraft release handles this scenario be used as a "control" or do the IRL pilots need to be consulted for what the behavior is in this situation? 

I'd have to check the places this function is called from, but replicating the current behavior would be a good starting point.

Reply:
OK Sounds good.